ts/src/index.ts

```diff
@@ -1,3 +1,6 @@
+import { Keypair, PublicKey, Transaction } from "@solana/web3.js";
+import type { Wallet as ProviderWallet } from "./provider";
```

Comment:
Do we need lines 1 and 2 here?

Reply:
ahh, my bad that lingered from testing locally.